style.css

```diff
@@ -223,8 +224,8 @@
   font-family: "Inter", sans-serif;
   font-style: normal;
   font-weight: 500;
-  font-size: 15px;
-  line-height: 20px;
+  font-size: 0.938rem;
+  line-height: 1.25rem;
   color: #091e42;
   flex: none;
```

Comment:
- [ ] I think it's better to use `flex-basis=content` to fix the alignment problem of this list. Your portfolio will look more professional.

Reply:
done